packages/shared/tsconfig.json

```diff
@@ -15,6 +15,10 @@
         },
         "types": ["node"]
     },
-    "include": ["./lib"],
+    "include": [
+        "./lib",
+        "../desktop/views/onboarding/views/create-profile/create-profile-route.enum.ts",
+        "../desktop/views/onboarding/views/create-profile/create-profile-router.ts"
```

Comment:
can we avoid this?
Do we want these routes to be included in shared, because the onboarding flow will be similar in desktop and mobile?

Reply:
Yeah this was an accident with auto import I will remove these